package.json

```diff
@@ -17,17 +17,17 @@
     "contributors:add": "all-contributors add",
     "contributors:generate": "all-contributors generate",
     "contributors:check": "all-contributors check",
-    "postinstall": "run-p install:*",
     "docs": "./scripts/docsite.sh",
+    "postinstall": "run-p -l --max-parallel 3 'install:*'",
```

Comment:
ℹ️  `-l` adds labels to the command, which I think helps clear up what the command does. Not related to the fix, just figured it looks nice.